src/lib.rs

```diff
@@ -206,6 +206,10 @@ pub use source_futures::*;
 mod thread_pool;
 pub use thread_pool::ThreadPool;
 
+/// This is the log domain used by the [`clone!`][crate::clone] macro. If you want to customize the
+/// message, you can set your own logger using the corresponding `log` functions.
```

Comment:
It's not about customizing the message. It's about having a custom logger that does not go to stdout but for example to syslog.